protocol/messages.py

```diff
@@ -10,10 +10,14 @@
 - ADD - add user to client list - [client >> server]
 - LST - get contact list from server - [client >> server]
 - CHG - change info about new status of users - [client >> server][server >> client]
-- ACK - acknowledgement message - [client >> server][server >> client]
+- ACK - acknowledgement message - [client >> server][server >> client]    
+- ACH - Add new participants to existent chat [client >> server]
+- GCH - Get list of available chats[clint >> server]
```

Comment:
Not a remark, but a useful proposal: we should support some event notifications to be able, for example, to send to user notifications that new participant was added/removed to chat

Reply:
Yes, I agree with you.